doc/rst/source/animations/anim13.rst

```diff
@@ -0,0 +1,18 @@
+.. _anim13:
+
+(13) Animating Time-Series
+==========================
+
+Here we demonstrate how one can animate a time-series so that the line grows in length and temporarily
+change pen thickness and color/intensity as it is being written.  The trick we use is to convert the
```

Comment:
```suggestion
Here we demonstrate how one can animate a time-series so that the line grows in length and the
pen thickness and color/intensity changes as it is being written.  The trick we use is to convert the
```